Backend/ServerManager.py

```diff
@@ -218,20 +218,20 @@ def api_get_trends_snapshot():
             until = datetime.now()
             since = until - timedelta(hours=12)
             woeid = 1
-            days = 1
-            hours = 0
-            minutes = 0
+            days = 0
+            hours = 3
+            minutes = 30
             seconds = 0
 
-            if fromdate is not None and todate is not None:
-                try:
-                    parsedfrom = datetime.strptime(fromdate, "%Y-%m-%d %H:%M:%S")
-                    parsedto = datetime.strptime(todate, "%Y-%m-%d %H:%M:%S")
-                    if(parsedfrom < parsedto):
-                        since = parsedfrom
-                        until = parsedto
-                except:
-                    print("/temporal -- datetimes not in the proper format of YYYY-mm-dd HH-MM-SS -- using default datetime")
+            try:
+                since = datetime.strptime(fromdate, "%Y-%m-%d %H:%M:%S")
+                until = datetime.strptime(todate, "%Y-%m-%d %H:%M:%S")
+            except:
+                print("/temporal -- datetimes not in the proper format of YYYY-mm-dd HH-MM-SS -- using default datetime")
```

Comment:
This isn't a good way to do this. The None-checks are good, you should just have separate code blocks (each with its own null/None-check) for fromdate and todate.
Right now, if someone provided a todate but not a fromdate, that would be ignored because the first line in the try block would fail from trying to pass None into datetime.strptime().